src/annotator/frame-observer.js

```diff
@@ -21,6 +22,14 @@ export default class FrameObserver {
     );
   }
 
+  /**
+   * Registers two listeners: the first callback is fired when an Hypothesis frame
+   * is added, the second when an Hypothesis frame is removed.
+   * This method is expected to be called only once.
+   *
+   * @param {(frame: HTMLIFrameElement) => void } onFrameAddedCallback
```

Comment:
```suggestion
   * @param {(frame: HTMLIFrameElement) => void} onFrameAddedCallback
```

With the improved JSDoc here the `Callback` suffix is now redundant IMO, so this could be simplified to `observe(onFrameAdded, onFrameRemoved)`.